spec/services/halls/delete_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Halls::Delete do
+  let!(:hall) { create :hall, name: 'name', capacity: 30 }
```

Comment:
```suggestion
  let!(:hall) { create :hall }
```

name and capacity has no special role in this test, you can use [defaults](https://github.com/danyflorczak/Monte_Cinema/pull/16/files#diff-e1e712fa419ca15713587465c1ad6a27f6235af650bdfe84b049f28574de068cR5-R6).